jenkinsfiles/dev

```diff
@@ -182,6 +182,41 @@ pipeline {
                 }
             }
         }
+
+        stage('Update IM Play dev instance') {
+            if (env.GIT_BRANCH == 'master') {
+                steps {
+                    script {
+                        withCredentials([usernamePassword(credentialsId: 'e2e-im-user', passwordVariable: 'PASSWORD', usernameVariable: 'USER_EMAIL')]) {
+                            dir('im-db-manager') {
+                                sparseCheckout('https://github.com/dhis2-sre/im-database-manager', 'master', '/scripts')
+
+                                dir('/scripts') {
```

Comment:
It should be just `scripts` here without the leading `/`.

Reply:
How come it worked the other place? It's your code that I stole ;-)

Reply:
It didn't :D You managed to copy it while it still was WIP and not tested